entity_history/models.py

```diff
@@ -21,7 +21,7 @@ class EntityRelationshipActivationEvent(models.Model):
     super_entity = models.ForeignKey(
         Entity, related_name='+', help_text='The super entity in the relationship that was activated / deactivated')
     time = models.DateTimeField(db_index=True, help_text='The time of the activation / deactivation')
-    was_activated = models.BooleanField(help_text='True if the entity was activated, false otherwise')
+    was_activated = models.BooleanField(default=False, help_text='True if the entity was activated, false otherwise')
```

Comment:
I forgot, but is there a reason these have to have defaults? I prefer that the user has to always enter a value for the field rather than it defaulting to something